programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -2709,20 +2707,20 @@ void scm_service_stop_event(uv_loop_t *loop, void *arg) {
     ZITI_LOG(INFO,"============================ service ends ==================================");
 
     if (main_ziti_loop != NULL) {
+
         uv_stop(main_ziti_loop);
-        uv_loop_close(main_ziti_loop);
+        int result = uv_loop_close(main_ziti_loop);
+        if (result == UV_EBUSY)
+        {
+            ZITI_LOG(INFO, "loop is busy, exiting...");
```

Comment:
In your experience, does this ever _not_ happen? I'm asking because if uv_loop_close always returns EBUSY and we just fall through here then it would probably be better to just fall through without trying to call uv_loop_close while on the SCM thread.

Reply:
yes, it always returns EBUSY. I will remove the call to uv_loop_close from the SCM thread